packages/compile-solidity/test/test_supplier.js

```diff
@@ -150,13 +150,10 @@ describe("CompilerSupplier", function () {
         Config.getTruffleDataDirectory(),
         "compilers/node_modules"
       );
-      const expectedCache = path.resolve(
-        compilerCacheDirectory,
-        "soljson-v0.4.21+commit.dfe3193c.js"
-      );
 
       // Delete if it's already there.
-      if (await fse.exists(expectedCache)) await fse.unlink(expectedCache);
+      if (await fse.exists(compilerCacheDirectory))
```

Comment:
Minor note here: `fs.exists` is deprecated.  You should probably use `fs.existsSync` instead.

Reply:
I'll go ahead and address these concerns in https://github.com/trufflesuite/truffle/pull/5445 as it moves the compile-solidity tests to a new package. It should make dealing with merge conflict stuff easier.

Reply:
Oh actually I will need to address these here I think since some of these are related to necessary test suite changes for this particular PR.